### PR TITLE
Add admin

### PR DESCRIPTION
**Pull request by Aafke89**
- Added the ActiveAdmin Gem
- As an admin you can
  -- Delete a post
  -- Delete a request
  -- Delete an user
  -- Make another user an admin

Things that are good to know:
-- As Request is a reserved name in Rails I got an undefined_method_error: user_agent. I know "solved" it by making an empty method in the Usermodel
-- It had some difficutlies with the messenger-id in the Requestshow of the admin, so I also made a custom method for that.

Ajeetoooo

### CODE REVIEW

File: app/admin/post.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# See permitted parameters documentation:
+# https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+
+
+
+# permit_params :subject, :content
+
+actions :all, except: [:edit, :update]
+
```
Comment (MerelSteenbrink) [inline app/admin/post.rb]: So the admin can create a new post?

File: app/admin/request.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+ActiveAdmin.register Request do
+
+# See permitted parameters documentation:
+# https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+#
+# permit_params :message, :status
+
+actions :all, except: [:edit, :update]
+
```
Comment (MerelSteenbrink) [inline app/admin/request.rb:10]: same: admin can make request?

File: app/admin/request.rb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+  actions
+end
+
+form do |f|
+    f.inputs "Request" do
+      f.input :message
+      f.input :status
+    end
```
Comment (MerelSteenbrink) [inline app/admin/request.rb:14]: We can leave this right as the user cannot edit a request right?

**Comment by Aafke89** [comment]
I updated the admin.
Now you can not make a new request, post or user in the adminboard. 
And an admin can not delete another admin.

**Pull request merged**